src/js/profile/blocks/indicator.js

```diff
@@ -31,4 +34,26 @@ export class Indicator extends ContentBlock {
 
         }
     }
+
+    orderChartData() {
+        let primaryGroup = this.indicator.metadata.primary_group;
+        let groupsOrder = this.indicator.metadata.groups.filter((g) => {
+            return g.name === primaryGroup
+        })[0];
+        let orderedGroups = [];
+
+        groupsOrder.subindicators.forEach((g) => {
+            let d = this.indicator.data.filter((x) => {
+                return x[primaryGroup] === g
+            })[0];
```

Comment:
Here 0 is returning first of the filtered list.
Fails if there is more than one data row.